samples/javascript_nodejs/51.teams-messaging-extensions-action/bots/teamsMessagingExtensionsActionBot.js

```diff
@@ -164,4 +173,57 @@ function shareMessageCommand(context, action) {
     };
 }
 
+function empDetails(context, action)
+{
+    return {
+        task: {
+          type: 'continue',
+          value: {
+            width: 350,
+            height: 300,
+            title: 'Task module WebView',
+            url: `${baseurl}/CustomForm.html`,
+          }
+        }
+    };
+}
+
+function dateTimeInfo(context, action)
+{
+    return {
+        task: {
+          type: 'continue',
+          value: {
+            width: 450,
+            height: 125,
+            title: 'Task module Static HTML',
+            url: `${baseurl}/StaticPage.html`,
+          }
+        }
+    };
+}
+
+async function webViewResponse(context, action) {
+    // The user has chosen to create a card by choosing the 'Create Card' context menu command.
+    const data = await action.data;
+    const heroCard = CardFactory.heroCard(`ID: ${data.EmpId}`, `E-Mail: ${data.EmpEmail}`);
+    heroCard.content.subtitle = `Name: ${data.EmpName}`;
+    const attachment = { 
+        contentType: heroCard.contentType, 
+        content: heroCard.content, 
+        preview: heroCard 
+    };
+    
+    return {
+        composeExtension: {
+              type: 'result',
+              attachmentLayout: 'list',
+              attachments: [
+                 attachment
+              ]
+        }
+    };
+}
```

Comment:
`context` is unused here as well as most of the other methods. Please review each method and remove unused variables and also remove them from where the function gets called.